packages/components/src/List/Content/Content.scss

```diff
@@ -156,7 +156,7 @@ $line-height: 30px;
 				font-weight: 400;
 				margin-right: $padding-small;
 				text-align: right;
-				text-transform: uppercase;
+				text-transform: capitalize;
```

Comment:
Gwendal wishes that Metadata names in "Expanded" mode must be in sentence case, not uppercase.

I talk with @maximenicoul about this to refresh the guidelines because now  Dataset Column Name (or Schema Object Name) must be in UpperCase